Marlin/src/lcd/menu/game/types.h

```diff
@@ -41,6 +49,130 @@ class MarlinGame {
   static bool game_frame();
   static void draw_game_over();
   static void exit_game();
+
 public:
   static void init_game(const uint8_t init_state, const screenFunc_t screen);
+
+  //
+  // Render API, based on U8GLib
+  // draw functions are implemented by the screen-specific renderer
+  //
+public:
+  /**
+   * @brief The colors available for drawing games.
+   * @note If a screen doesn't support (a) color, it shall fall back to using WHITE.
```

Comment:
update note: "if a screen doesn't support (a) color, it is expected to map to the closes color or white if the closest color is (near) black.

can also remove the comments on color enum values